exchange/adapter_util.go

```diff
@@ -118,6 +118,7 @@ func GetDisabledBidderWarningMessages(infos config.BidderInfos) map[string]strin
 		"groupm":         `Bidder "groupm" is no longer available in Prebid Server. Please update your configuration.`,
 		"verizonmedia":   `Bidder "verizonmedia" is no longer available in Prebid Server. Please update your configuration.`,
 		"brightroll":     `Bidder "brightroll" is no longer available in Prebid Server. Please update your configuration.`,
+		`engagebdr`:      `Bidder "engagebdr" is no longer available in Prebid Server. Please update your configuration.`,
```

Comment:
should use `"` double quotes for engagebdr key

Reply:
addressed here - https://github.com/prebid/prebid-server/pull/3131/commits/1870aed0b3935a8acd0333a6a29279ac4ce076c5